build.gradle

```diff
@@ -306,9 +310,13 @@ dependencies {
     exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
   }
 
-  testImplementation 'org.mockito:mockito-inline:3.5.10'
+  testImplementation group: 'org.camunda.bpm.extension.springboot', name: 'camunda-bpm-spring-boot-starter-test', version: '2.2.0'
+
+  testImplementation group: 'com.h2database', name: 'h2', version: '1.4.200'
```

Comment:
is this used anywhere?

Reply:
h2 is requried for createStandaloneInMemProcessEngineConfiguration on line 33 of BmpnBaseTest

https://github.com/hmcts/unspec-service/blob/d6b870374a9a2a195817ca23bcb25717ddaff512/src/test/java/uk/gov/hmcts/reform/unspec/bpmn/BpmnBaseTest.java#L33

Diving into the code `this.jdbcUrl = "jdbc:h2:mem:camunda";`